app/controllers/api/v1/api_controller.rb

```diff
@@ -28,6 +28,12 @@ def status
       def render_error(exception)
         raise exception if Rails.env.test?
 
+        # To properly handle RecordNotFound errors in views
+        if exception.message.starts_with?("Couldn't find")
```

Comment:
Can't we do something like `if exception.is_a?(RecordNotFound)` instead?

Reply:
As far as I know, no. The exception's class is `ActionView::Template::Error`

Reply:
Correction: as we discussed, we can use `cause`. Fixed.